kodilla-code-coverage/src/main/java/com/kodilla/hibernate/invoice/Product.java

```diff
@@ -36,7 +36,7 @@ public String getName() {
     @OneToMany(
             targetEntity = Item.class,
             mappedBy = "product",
-            cascade = CascadeType.ALL,
+            cascade = CascadeType.PERSIST,
```

Comment:
Nie, chodzi o usuniecie calkowite Cascade z Product. Cascade PERSIST mowi o tym, ze kiedy zapisujesz nowy Product, to razem z nim powinny byc zapisane